addons/website/static/tests/tours/snippet_rating.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module **/
+import wTourUtils from '@website/js/tours/tour_utils';
+
+wTourUtils.registerWebsitePreviewTour('snippet_rating', {
```

Comment:
Use double quotes everywhere please, It's our new convention, thanks

Reply:
next commit I made the changes related to double quotes wherever it is possible.
note that it is not possible in this case:`'we-button[data-select-class="s_rating_no_title"]', 'we-button[data-select-class="s_rating_inline"]', 'we-button[data-select-class=""]' ` 
thanks.